api/python/t4/packages.py

```diff
@@ -527,6 +527,13 @@ def push(self, name, path):
             A new package that points to the copied objects
         """
         dest = fix_url(path)
+        if dest.startswith('file://'):
+            # local push
+            os.makedirs(path, exist_ok=True)
+            pkg = self._materialize(dest)
+            pkg.build(name)
+            return pkg
+
         if not dest.startswith('s3://'):
```

Comment:
Probably should change the flow of these conditionals for readability. Possibly:

```
if 'file://':
...
elif 's3://' ''
...
else
raise NIE
```